framework/src/media/audio/audio_manager.cpp

```diff
@@ -2903,3 +2903,64 @@ void dump_audio_card_info()
 	print_audio_card_info(OUTPUT);
 }
 #endif
+
+audio_manager_result_t start_audio_multich_stream(int card_id, int device_id, int duration, int verbose, uint32_t *dev_extract_size)
+{
+    char path[AUDIO_DEVICE_FULL_PATH_LENGTH];
+	audio_card_info_t *card;
+    audio_manager_result_t ret;
```

Comment:
You should use a tab for indentation instead of spaces.
Several lines have spaces.